[2025-03-20T01:10:35.630+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-20T01:10:35.681+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: welcome_dag.print_date scheduled__2025-03-18T23:00:00+00:00 [queued]>
[2025-03-20T01:10:35.715+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: welcome_dag.print_date scheduled__2025-03-18T23:00:00+00:00 [queued]>
[2025-03-20T01:10:35.717+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-20T01:10:35.779+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): print_date> on 2025-03-18 23:00:00+00:00
[2025-03-20T01:10:35.792+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'welcome_dag', 'print_date', 'scheduled__2025-03-18T23:00:00+00:00', '--job-id', '109', '--raw', '--subdir', 'DAGS_FOLDER/welcome_dag.py', '--cfg-path', '/tmp/tmpb_9ykj4v']
[2025-03-20T01:10:35.797+0000] {standard_task_runner.py:105} INFO - Job 109: Subtask print_date
[2025-03-20T01:10:35.804+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=169) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-20T01:10:35.812+0000] {standard_task_runner.py:72} INFO - Started process 192 to run task
[2025-03-20T01:10:35.959+0000] {task_command.py:467} INFO - Running <TaskInstance: welcome_dag.print_date scheduled__2025-03-18T23:00:00+00:00 [running]> on host e3789c96a6c5
[2025-03-20T01:10:36.188+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='welcome_dag' AIRFLOW_CTX_TASK_ID='print_date' AIRFLOW_CTX_EXECUTION_DATE='2025-03-18T23:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-18T23:00:00+00:00'
[2025-03-20T01:10:36.191+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-20T01:10:36.219+0000] {logging_mixin.py:190} INFO - Today is 2025-03-20
[2025-03-20T01:10:36.221+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-20T01:10:36.250+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-20T01:10:36.251+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=welcome_dag, task_id=print_date, run_id=scheduled__2025-03-18T23:00:00+00:00, execution_date=20250318T230000, start_date=20250320T011035, end_date=20250320T011036
[2025-03-20T01:10:36.392+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-20T01:10:36.494+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-20T01:10:36.538+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
